src/witan/models/dem/ccm/fert/hist_asfr_ws.clj

```diff
@@ -1,41 +1,159 @@
 (ns witan.models.dem.ccm.fert.hist-asfr-ws
   (:require [witan.models.dem.ccm.fert.hist-asfr-age :as asfr]
             [witan.workspace-executor.core :as wex]
-            [schema.core :as s]))
+            [schema.core :as s]
+            [witan.models.load-data :as ld]))
 
 ;; GET DATA FOR ROOT NODES
+(defn get-data
+  [keyname]
+  (let [test-data-paths {:births-data "resources/test_data/bristol_births_data.csv"
+                         :at-risk-popn "resources/test_data/bristol_denominators.csv"
+                         :mye-coc "resources/test_data/bristol_mye_coc.csv"}
+        data-map (ld/load-datasets test-data-paths)]
+    (get data-map keyname)))
```

Comment:
I'd invert this logic slightly so that you're not loading all the datasets and then only selecting one each time you run `get-data`.